### PR TITLE
Update Visual Studio 2022 support

### PR DESCRIPTION
**Pull request by sharwell**

### CODE REVIEW

File: Tvl.VisualStudio.MouseFastScroll/source.extension.vsixmanifest [hunk 1]
```diff
@@ -14,13 +14,26 @@
...
+    </InstallationTarget>
+    <InstallationTarget Version="[17.0,18.0)" Id="Microsoft.VisualStudio.Community">
+      <ProductArchitecture>amd64</ProductArchitecture>
+    </InstallationTarget>
+    <InstallationTarget Version="[17.0,18.0)" Id="Microsoft.VisualStudio.Pro">
+      <ProductArchitecture>x86</ProductArchitecture>
+    </InstallationTarget>
+    <InstallationTarget Version="[17.0,18.0)" Id="Microsoft.VisualStudio.Pro">
+      <ProductArchitecture>amd64</ProductArchitecture>
+    </InstallationTarget>
```
Comment (AArnott) [inline Tvl.VisualStudio.MouseFastScroll/source.extension.vsixmanifest:30]: Why do you need Pro here? Targeting Community allow it to work on higher SKUs.

Comment (sharwell) [reply Tvl.VisualStudio.MouseFastScroll/source.extension.vsixmanifest:30]: I'm guessing I don't need it separated for 17+. There were some earlier versions where Community wasn't recognized.

Comment (AArnott) [reply Tvl.VisualStudio.MouseFastScroll/source.extension.vsixmanifest]: 16.0 didn't need it either AFAIK. I think we only needed special handling like this back when we had the Express SKUs.

**Review by AArnott — APPROVED** [review]

File: appveyor.yml [hunk 2]
```diff
@@ -28,19 +28,21 @@ test_script:
...
+    #    -output:OpenCover.MouseFastScroll.IntegrationTests.xml `
+    #    -target:"$env:xunit20\xunit.console.x86.exe" `
+    #    -targetargs:"$integrationTestAssembly -appveyor -noshadow"
+    #if ($LastExitCode -ne 0) { $host.SetShouldExit($LastExitCode) }
     &$codecov -f OpenCover.MouseFastScroll.UnitTests.xml --flag unit
-    &$codecov -f OpenCover.MouseFastScroll.IntegrationTests.xml --flag integration
+    #&$codecov -f OpenCover.MouseFastScroll.IntegrationTests.xml --flag integration
 artifacts:
 - path: 'Tvl.VisualStudio.MouseFastScroll\**\*.vsix'
+- path: '\msbuild.binlog'
```
[the remark this answers is not in the document]
Comment (sharwell) [reply appveyor.yml:47]: Yes the path works, but I'm not sure it's in the root directory.

[the remark this answers is not in the document]
Comment (sharwell) [reply appveyor.yml:47]: ➡️ According to the binlog itself, the path is:

> C:\\projects\\mousefastscroll\\msbuild.binlog

File: Tvl.VisualStudio.MouseFastScroll/source.extension.vsixmanifest (hunk 1 above)
Comment (sharwell) [reply Tvl.VisualStudio.MouseFastScroll/source.extension.vsixmanifest]: This extension (like many of my extensions) supports Visual Studio 2012 and newer.

**Pull request merged**